pkg/identity/identitysync/registry.go

```diff
@@ -57,6 +58,11 @@ func Registry() IdentityProviderRegistry {
 				Description:      "AWS Single Sign On",
 				DocsID:           "aws-sso",
 			},
+			IDPTypeOneLogin: {
```

Comment:
Can you please add an integration test for the OneLogin sync method? You can set it up to call your dev OneLogin instance and just assert that we found some users and groups